migrations/20221031220419_alter_request_add_indexes.js

```diff
@@ -0,0 +1,31 @@
+const REQUEST_TABLE_NAME = 'request'
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export async function up(knex) {
```

Comment:
what about indexes for the anchor table?  We query anchors by requestId so should probably have an index on requestId.  Also should probably have one on anchor cid while we're at it

Reply:
we don't have any queries using anchor cid, but i'll add one for requestId

Reply:
also we do not need one for requestId because this column is unique. Postgres auto makes unique indexes for primary keys and unique columns. 

The note in their docs: There's no need to manually create indexes on unique columns; doing so would just duplicate the automatically-created index (https://www.postgresql.org/docs/current/indexes-unique.html)